velox/serializers/PrestoSerializer.cpp

```diff
@@ -689,15 +646,15 @@ void readColumns(
     ByteStream* source,
     velox::memory::MemoryPool* pool,
     const std::vector<TypePtr>& types,
-    std::vector<VectorPtr>* result,
+    std::vector<VectorPtr>& result,
     bool useLosslessTimestamp) {
   static std::unordered_map<
       TypeKind,
       std::function<void(
           ByteStream * source,
           std::shared_ptr<const Type> type,
           velox::memory::MemoryPool * pool,
-          VectorPtr * result,
+          VectorPtr & result,
```

Comment:
nit: Is it formatted? Thanks!

Reply:
It does look strange indeed. I assume this is a bug in auto-formatter.